requirements.txt

```diff
@@ -1,2 +1,3 @@
 jupyterlab==4.3.3
 notebook==7.3.1
+python-dotenv~=1.0.1
```

Comment:
I don't think this is needed because we do it in the notebook already, but it doesn't hurt.